src/test/java/seedu/address/logic/parser/TakeAttendanceCommandParserTest.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.TakeAttendanceCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.attendance.AttendancePresence;
+import seedu.address.model.person.Name;
+import seedu.address.model.session.SessionNumber;
+
+public class TakeAttendanceCommandParserTest {
```

Comment:
Just a nit, but perhaps you could also test for invalid inputs for the sessions, names or attendance presence?